PyPDF2/generic/_base.py

```diff
@@ -66,6 +79,36 @@ def hash_value(self) -> bytes:
             )
         ).encode()
 
+    def clone(
+        self,
+        pdf_dest: "PdfWriter",  # type: ignore
```

Comment:
```suggestion
        pdf_dest: "PdfWriter",
```

Reply:
are you sure this is OK for old version?

Reply:
Oh, good question. I only checked locally with Python 3.10.2.

Let me upload my changes in another branch to verify (I'll delete that branch later)

Reply:
https://github.com/py-pdf/PyPDF2/pull/1396

Reply:
Seems to be ok :-)